pure.zsh

```diff
@@ -227,9 +227,9 @@ prompt_pure_async_callback() {
 			prompt_pure_git_dirty=$output
 			prompt_pure_preprompt_render
 
-			# when prompt_pure_git_delay_dirty_check is set, the git info is displayed in a different color, this is why the
-			# prompt is rendered before the variable is (potentially) set
-			(( $exec_time > 2 )) && prompt_pure_git_delay_dirty_check=$EPOCHSECONDS
+			# when prompt_pure_git_last_dirty_check_timestamp is set, the git info is displayed in a different color,
+			# this is why the prompt is rendered before the variable is (potentially) set
```

Comment:
@mafredri Hmm, but I think it is? `git_color` will be `red` if this variable is set (see https://github.com/zmwangx/pure/blob/5b522d86dad019b08e2684114fc9716266d2ffc6/pure.zsh#L85-L86).